src/app/dashboard/plugin-config.html

```diff
@@ -1,13 +1,15 @@
-<md-dialog>
+<md-dialog flex="30">
   <md-dialog-content>
-    <!-- {{vm.plugin.title}} -->
+    <h2>{{vm.plugin.title}}</h2>
```

Comment:
Hmm, I think it's only important during the adding phase, because at that time you want reference to the thing you're adding. But during editing, the context is flipped to you want reference to the thing you're editing.